src/app/dev/DevToys.Api/Tool/GUI/Components/IUICard.cs

```diff
@@ -0,0 +1,89 @@
+﻿namespace DevToys.Api;
+
+/// <summary>
+/// A component that represents a setting, with a title, description, icon and <see cref="IUIElement"/> for the option value.
+/// </summary>
+public interface IUICard : IUIElementWithChildren
+{
+    /// <summary>
+    /// Gets the list of child elements.
+    /// </summary>
+    IUIElement[]? Children { get; }
+
+    /// <summary>
+    /// Raised when <see cref="Children"/> is changed.
+    /// </summary>
+    event EventHandler? ChildrenChanged;
+}
+
+[DebuggerDisplay($"Id = {{{nameof(Id)}}}")]
+internal class UICard : UIElementWithChildren, IUICard
+{
+    private IUIElement[]? _children;
+
+    internal UICard(string? id)
+        : base(id)
+    {
+    }
+
+    protected override IEnumerable<IUIElement> GetChildren()
+    {
+        if (_children is not null)
+        {
+            foreach (IUIElement child in _children)
+            {
+                if (child is not null)
+                {
+                    yield return child;
+                }
+            }
+        }
+    }
+
+    public IUIElement[]? Children
+    {
+        get => _children;
+        internal set => SetPropertyValue(ref _children, value, ChildrenChanged);
+    }
+
+    public event EventHandler? ChildrenChanged;
+}
+
+public static partial class GUI
+{
+    /// <summary>
+    /// A component that represents a setting, with a title, description, icon and <see cref="IUIElement"/> for the option value.
+    /// </summary>
+    public static IUICard Card()
+    {
+        return Card(null);
+    }
+
+    /// <summary>
+    /// A component that represents a setting, with a title, description, icon and <see cref="IUIElement"/> for the option value.
+    /// </summary>
+    /// <param name="id">An optional unique identifier for this UI element.</param>
+    public static IUICard Card(string? id)
+    {
+        return new UICard(id);
+    }
+
+    /// <summary>
+    /// Sets the <see cref="IUISetting.InteractiveElement"/> of the setting.
+    /// </summary>
+    public static IUICard InteractiveElement(this IUICard element, IUIElement? uiElement)
+    {
+        ((UISetting)element).InteractiveElement = uiElement;
+        return element;
+    }
+
+    /// <summary>
+    /// Sets the children to be displayed in the stack.
+    /// </summary>
+    public static IUICard WithChildren(this IUICard element, params IUIElement[] children)
```

Comment:
I do see that by default, if I put multiple items in this method, items are being stacked horizontally (despite the lack of Stack Panel, but CSS makes it happen this way):

![image](https://github.com/veler/DevToys/assets/3747805/594435ad-2c6f-419d-b67c-23bd73e4d1b0)

In my opinion, this component should not assume to stack items vertically. Instead of accepting a list of children, I think it should accept a single child `IUIElement`, which can be anything from a button to a stack for example. That would be consistent with other similar containers that aren't explicitly made to support multiple items. For example, `UIDialog`, `IUIListItem`, `IUIGridCell`, `IUIDataGridCell`, `IUISetting.InteractiveElement` (there's an example in Settings tool where the InteractiveElement is a Stack), `IUIDataGrid.CommandBarExtraContent` (which is also a Stack in JSON > Table tool).